circuits/circom/test/membership_test.circom

```diff
@@ -0,0 +1,5 @@
+pragma circom 2.0.2;
+
+include "../membership.circom";
+
+component main {public [msghash, root]} = InAddressSet(64, 4, 2);
```

Comment:
This is the entrypoint for proving set membership. The message hash and the merkle root of addresses are the only public inputs.